### ISSUE TITLE
Unintended Uninstall of Windows Apps

### ISSUE THREAD

**Issue by CraigAT** [microsoft/windows-dev-box-setup-scripts#94]
I found this script and thought it would be the ideal way to set up my computer with Visual Studio to do some coding.  It was not clear by running the full script a lot of Windows 10 apps would be uninstalled permanently (e.g Minecraft, Xbox app).  Is there as simple script to reinstall these or a way to use this script to reinstall them?

**Comment by p1-ra** [comment]
I agree, I fall to that script from Microsoft documentation, I thought it was safe and only did what it said, install "Full Desktop App" env.
This is a damn **** script (sorry), but I just saw it uninstalling a lot of my app and got a notification about **UAC trying to be disabled**.

![image](https://user-images.githubusercontent.com/18233250/132999613-eb6a1fb7-2b7f-49e9-88bd-c5b32861621e.png)

That absolutely not acceptable, will never trust Microsoft documentation external links anymore (btw, came from https://docs.microsoft.com/en-us/learn/modules/introduction-to-windows10-dev/2-installing-tools-automatically)

Waste my time, now I need to review any accounts perform by the script.
